test/e2e/warnings_test.go

```diff
@@ -72,27 +81,62 @@ spec:
   image: my-awesome-cron-image
 `
 
+	expectedOutputTemplate := `
+Target cluster 'https://127.0.0.1:60817' (nodes: minikube)
```

Comment:
port and nodes would be different probably so dont think we can keep this info.

Reply:
We have a function `replaceTarget()` which removes the entire target string, we use this both for expected output and the output itself. We could exclude it in the expected output and call `replaceTarget()` only for the output itself